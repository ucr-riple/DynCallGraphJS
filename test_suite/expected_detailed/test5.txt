{
    'get foo (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test5.js:9:1:9:8)': [
        'get foo (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test5.js:4:13:6:4)'
    ]
}