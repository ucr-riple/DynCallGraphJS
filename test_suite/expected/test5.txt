{
  '(test5.js:9:1:9:8)': [
    '(test5.js:4:13:6:4)'
  ]
}